eras/conway/impl/testlib/Test/Cardano/Ledger/Conway/ImpTest.hs

```diff
@@ -362,6 +391,14 @@ submitProposal ::
   ImpTestM era (GovActionId (EraCrypto era))
 submitProposal proposal = trySubmitProposal proposal >>= expectRightExpr
 
+submitProposal_ ::
+  ( ShelleyEraImp era
+  , ConwayEraTxBody era
+  ) =>
+  ProposalProcedure era ->
+  ImpTestM era ()
+submitProposal_ = void . submitProposal
```

Comment:
Great minds think alike :wink:  I also added this function in one of my PRs :slightly_smiling_face:

Reply:
Yeah, I noticed that as well :smile: